src/main/java/com/friendly_machines/intellij/plugins/ideanative2debugger/impl/GdbMiFilter.java

```diff
@@ -110,7 +110,10 @@ public GdbMiStateResponse gdbSend(String operation, Collection<String> options,
 
     public Map<String, ?> gdbCall(String operation, Collection<String> options, Collection<String> parameters) throws GdbMiOperationException, IOException, InterruptedException {
         var response = gdbSend(operation, options, parameters);
-        assert response.getMode() == '^';
+        if (response.getMode() != '^') {
+            // pucgenie: I don't like that repacking just for adding an error message. Data is lost too (see com.friendly_machines.intellij.plugins.ideanative2debugger.impl.GdbMiStateResponse#errorResponse ).
```

Comment:
I think it's fine in this case to repack again since that should never be reached anyway.